msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md

```diff
@@ -124,8 +124,131 @@ For the complete app manifest, see [App manifest schema](~/resources/schema/mani
 |Teams messaging extension action| Describes how to define action commands, create task module, and  respond to task module submit action. |[View](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/csharp_dotnetcore/51.teams-messaging-extensions-action)|[View](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/javascript_nodejs/51.teams-messaging-extensions-action) | 
 |Teams messaging extension search   |  Describes how to define search commands and respond to searches.        |[View](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/csharp_dotnetcore/50.teams-messaging-extensions-search)|[View](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/javascript_nodejs/50.teams-messaging-extensions-search)|
 
+## Universal Actions for search based messaging extensions
+
+The Universal Actions now bring the enhanced user experience available for adaptive cards to messaging extensions. To enable Universal Actions for search based messaging extensions, the app must have the [Schema for Universal Actions](/adaptive-cards/authoring-cards/universal-action-model#schema) along with the following requirements:
```

Comment:
Instead of 
> the app must have the [Schema for Universal Actions](/adaptive-cards/authoring-cards/universal-action-model#schema) along with the following requirements
I suggest 
> the app must conform to the [Universal Actions' schema](/adaptive-cards/authoring-cards/universal-action-model#schema) along with the following requirements

Reply:
We have followed the standard format of giving the article name for the links.